modules/apps/27-gmp/types/account_test.go

```diff
@@ -0,0 +1,282 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/cosmos/gogoproto/proto"
+	"github.com/stretchr/testify/require"
+
+	sdkmath "cosmossdk.io/math"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+
+	gmp "github.com/cosmos/ibc-go/v10/modules/apps/27-gmp"
+	"github.com/cosmos/ibc-go/v10/modules/apps/27-gmp/types"
+	host "github.com/cosmos/ibc-go/v10/modules/core/24-host"
+	ibcerrors "github.com/cosmos/ibc-go/v10/modules/core/errors"
+)
+
+func TestBuildAddressPredictable(t *testing.T) {
+	testCases := []struct {
+		name      string
+		accountID *types.AccountIdentifier
+		expErr    error
+	}{
+		{
+			"success: valid account identifier",
+			&types.AccountIdentifier{
+				ClientId: "07-tendermint-0",
```

Comment:
You can use the FirstClientID instead of "07-tendermint-0". Ditto in rest of this file